packages/rum-core/src/domain/contexts/viewContexts.ts

```diff
@@ -39,6 +47,7 @@ export function startViewContexts(lifeCycle: LifeCycle): ViewContexts {
       version: view.version,
       id: view.id,
       name: view.name,
+      documentVersion: 0,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about also adding `documentVersion` to the `ViewCreatedEvent`? 
it would avoid to duplicate its initialization logic